source/getting-started/index.markdown

```diff
@@ -62,8 +62,10 @@ We will need a few things to get started with installing Home Assistant. The Ras
 5. Insert the SD card into your Raspberry Pi. If you are going to use an Ethernet cable, connect that too.
 6. Connect your power supply to the Raspberry Pi.
 7. The Raspberry Pi will now boot up, connect to the Internet and download the latest version of Home Assistant. This will take about 20 minutes.
-8. Home Assistant will be available at `http://homeassistant.local:8123`. If you are running an older Windows version or have a stricter network configuration, you might need to access Home Assistant at `http://homeassistant:8123` or `http://X.X.X.X:8123` (replace `X.X.X.X` with your Pi's IP address).
-9. If you used a USB stick for configuring the network, you can now remove it.
+8. When this is complete, a login prompt will display on the command line. Ignore this, or refer to [console access](https://www.home-assistant.io/hassio/commandline/) for further information.
```

Comment:
This is confusing IMHO. There is nothing in this getting started guide that indicated one should connect a monitor. The steps above even explicitly state to connect the ethernet cable and power supply.

Adding this, might:
A) Make people think they need to connect to a screen/keyboard (which they should not)
B) Do follow the steps above and do not see the command line described here.

Reply:
Makes sense-- what about I...
* Strike changes to 8
* State that the user _doesn't_ need to connect a monitor or keyboard to the pI around lines 5 or 6
* Retain changes that clarify "on a separate machine...?"

Reply:
I'm sorry. I have issues visualizing your suggestion.
I also don't see the issue you are trying to solve. This is the first time I see this issue raised, in the past 3 years, on any of our media.

Reply:
I found myself stuck for a few minutes once the Pi booted up (is the CLI prompt a new feature that hasn't been documented yet? There's no note on this...this appears different than the few other Pi installs I've done...I guess I'll give it a shot on another machine...)

The intent here is that I'd save someone the few minutes of head-scratching I had, but if it hasn't come up in forums or elsewhere, then agreed, no need to fix what isn't broken :]